src/circuit/circuit_vanilla.rs

```diff
@@ -0,0 +1,922 @@
+//! The Orchard Action circuit implementation.
```

Comment:
Fromm @PaulLaux:
we need to make sure that this file is created and renamed from the original circuit file.
Instead of diff we need to see "File renamed without changes" or just a small diff.